package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "rlbotjavascriptexample",
+  "version": "1.0.0",
+  "description": "An example bot implemented in JavaScript",
+  "main": "main.js",
+  "license": "ISC",
+  "dependencies": {
+    "rlbot-test": "github:SuperVK/RLBotJS"
```

Comment:
I think this was maybe already discussed, but: Having the RLBotJS framework code in */RLBotJS as opposed to RLBot/RLBot creates some problems:
- When we make a change to the flatbuffer spec, it will be necessary for somebody to copy over https://github.com/SuperVK/RLBotJS/blob/master/rlbot/rlbot_generated.js into the separate repo.
- Easier to forget about JS when doing such updates. Currently we can have a single PR to RLBot/RLBot that includes any necessary changes to the python, Java, .NET frameworks all at once, and somebody can point out if one has been missed.
- Easier to make breaking changes to the JS framework by mistake. A maintainer will typically search the RLBot/RLBot repo for usages of a particular function when trying to avoid breaking changes.
- RLBot_Core_Interface.dll starts getting distributed via different channels (pypi and npm), and could get out of sync in very confusing ways.

Are there any serious obstacles to putting the JS framework inside RLBot/RLBot?

Reply:
Nope, it is definitely the plan to put the final package into the RLBot/RLBot repo, that's why I also realised I probably should have made this pull request after we have a stable RLBotJS release (which is installed from RLBot/RLBot).
So SuperVK/RLBotJS is really just a test repo, that's also why it's called rlbot-test atm.